website/source/docs/concepts/policies.html.md

```diff
@@ -99,6 +111,31 @@ capabilities. These mappings are as follows:
 
   * `read` - `["read", "list"]`
 
+## Permissions
+
+Paths offer an optional `permissions` field for fine-grain control over the
```

Comment:
This is no longer true (not a permissions field anymore, just two top-level values).